src/web/js/dashboard/Student.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Icon, Tooltip } from 'react-mdl';
+import { Card, CardTitle, CardText, Textfield, Button } from 'react-mdl';
+
+class Student extends Component {
+  state = {
+    id: this.props.details.id,
+    firstName: this.props.details.firstName,
+    lastName: this.props.details.lastName,
+    email: this.props.details.email,
+    editing: false
+  };
+  api = this.props.api;
+
+  handleClickEditStudent = () => {
+    this.setState({editing: true});
+  }
+
+  handleChange = (event) => {
+    this.setState({[event.target.id]: event.target.value});
+  }
+
+  handleClickRemoveStudent = () => {
+    if (confirm('Remove this student from the roster? This cannot be undone.')) {
```

Comment:
We can also move over to this in the future that's more in the spirit of the rest of the UI: https://react-mdl.github.io/react-mdl/components/dialog/